tests/base_classes.py

```diff
@@ -88,14 +90,18 @@ def get_data(self, *args, **kwargs):
         `artifactory_helpers/get_bigdata()`.
         This will then return the full path to the local copy of the file.
```

Comment:
I would add a comment here indicating the use of astroquery from an archive versus obtaining the data from Artifactory or the local store.

Reply:
Addressed with e77079e.